spec/models/official_postcode_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'spec_helper'
+
+describe OfficialPostcode, type: :model do
+
+  describe "is_valid_postcode?" do
+    let(:postcode) { create(:official_postcode).postcode }
```

Comment:
This let block is only used in 1 test - is it worth having as a let block rather than inside the test itself ?